src/Symfony/Component/Cache/Traits/FilesystemTrait.php

```diff
@@ -23,6 +23,33 @@
     use FilesystemCommonTrait;
 
     /**
+     * @param int|null $time
+     *
+     * @return int
+     */
+    public function prune($time = null)
```

Comment:
Figured it allows you to prune cache entries up to a specific time (if other than the current time). Someone may want to "removal all entries expiring in the next hour" or whatever. Do you not see any value in this?